frontend/talentsearch/src/js/components/search/SearchForm.tsx

```diff
@@ -33,6 +33,10 @@ function mapIdToValue<T extends { id: string }>(objects: T[]): Map<string, T> {
   }, new Map());
 }
 
+const bold = (msg: string) => {
```

Comment:
I recommend using `strong` in `frontend/common/src/helpers/format.tsx` or adding this function there.